wmn-data.json

```diff
@@ -5625,6 +5733,18 @@
         "cat" : "XXXPORNXXX",
         "valid" : true
        },
+       {
+        "name" : "Teespring",
+        "uri_check" : "https://commerce.teespring.com/v1/stores?slug={account}",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "sellerToken",
+        "m_code" : 404,
+        "m_string" : "errors",

```

Comment:
Please change this from the word "errors" which could appear in a valid page to something else like `\"errors\"` or `\"not found\"`